homeassistant/components/github/sensor.py

```diff
@@ -30,6 +30,11 @@
 ATTR_OPEN_PULL_REQUESTS = "open_pull_requests"
 ATTR_PATH = "path"
 ATTR_STARGAZERS = "stargazers"
+ATTR_FORKS = "forks"
+ATTR_CLONES = "clones"
+ATTR_CLONES_UNIQUES = "clones_uniques"
```

Comment:
Maybe this should be `clones_unique`?

Reply:
True, that would roll off the tongue better.